src/components/page-scheduled-job/index.js

```diff
@@ -33,99 +34,93 @@ const PageScheduledJob = (props) => {
+          <div className="grid grid--gap-medium grid--overview-columns">
+            <div className="grid grid--gap-medium">
+              <Typography>
+                Scheduled job{' '}
+                <strong>{smallScheduledJobName(scheduledJobName)}</strong>, job{' '}
+                <strong>{jobComponentName}</strong>
+              </Typography>
+              <StatusBadge type={scheduledJobStatus}>
+                {scheduledJobStatus}
+              </StatusBadge>
+            </div>
+            {scheduledJob && (
+              <div className="grid grid--gap-medium">
+                <Typography>
+                  Created{' '}
+                  <strong>
+                    <RelativeToNow time={scheduledJob.created} />
+                  </strong>
+                </Typography>
+                <Typography>
+                  Started{' '}
+                  <strong>
+                    <RelativeToNow time={scheduledJob.started} />
+                  </strong>
+                </Typography>
+                {scheduledJob.ended && (
+                  <>
+                    <Typography>
                       Ended{' '}
                       <strong>
-                        <RelativeToNow
-                          time={scheduledJob.ended}
-                        ></RelativeToNow>
+                        <RelativeToNow time={scheduledJob.ended} />
                       </strong>
-                    </p>
-                    <p>
+                    </Typography>
+                    <Typography>
                       Duration{' '}
                       <strong>
                         <Duration
                           start={scheduledJob.started}
                           end={scheduledJob.ended}
                         />
                       </strong>
-                    </p>
-                  </div>
+                    </Typography>
+                  </>
                 )}
-                <p>
-                  Status <ScheduledJobStatus status={scheduledJobStatus} />
-                </p>
-                {scheduledJobLog && (
-                  <p>
-                    <h2 className="o-heading-section">Log</h2>
-                    <AsyncResource asyncState={pollLogsState}>
-                      {scheduledJobLog && <Code copy>{scheduledJobLog}</Code>}
-                    </AsyncResource>
-                  </p>
-                )}
-                {!scheduledJobLog && <p>No logs</p>}
-              </section>
-            </div>
-          </React.Fragment>
-        </AsyncResource>
-      </main>
-    </React.Fragment>
+              </div>
+            )}
+          </div>
+        </section>
+        <section className="scheduled-job__log grid grid--gap-medium">
+          <Typography variant="h4">Log</Typography>
+          {scheduledJobLog ? (
+            <AsyncResource asyncState={pollLogsState}>
+              <Code copy>{scheduledJobLog}</Code>
```

Comment:
I don't know how long a log like this would usually be, but if it gets really long it may be nice with a download option.

Reply:
added download option